public/pages/ChangePolicy/components/ChangeManagedIndices/ChangeManagedIndices.tsx

```diff
@@ -109,6 +109,7 @@ export default class ChangeManagedIndices extends Component<ChangeManagedIndices
               // @ts-ignore
               onChange={this.props.onChangeManagedIndices}
               onSearchChange={this.onManagedIndexSearchChange}
+              compressed={useUpdatedUX ? true : false}
```

Comment:
```suggestion
              compressed={useUpdatedUX ? true : undefined}
```
undefined is preferred, and i just feel that it is totally OK to use a compressed ComboBox no matter the toggle is on or off.